oriedita-ui/src/main/java/oriedita/editor/action/BackColorAction.java

```diff
@@ -25,10 +30,29 @@ public BackColorAction() {
     @Override
     public void actionPerformed(ActionEvent e) {
         //以下にやりたいことを書く
-        Color backColor = JColorChooser.showDialog(frameProvider.get(), "B_col", Color.white);
+        Color backColor = showCustomColorDialog(frameProvider, "B_col", FlatLaf.isLafDark() ? Colors.FIGURE_BACK_DARK : Colors.FIGURE_BACK);
 
         if (backColor != null) {
             foldedFigureModel.setBackColor(backColor);
         }
     }
+
+    private Color showCustomColorDialog(FrameProvider frameProvider, String title, Color initialColor){
```

Comment:
this method is repeated multiple times, i think it would be useful to have it as a static helper method somewhere